HEAL-notebooks/Dockerfile

```diff
@@ -7,6 +8,7 @@ COPY $NOTEBOOK_DIR/ $HOME/
 RUN conda config --append channels conda-forge
 RUN conda install -c plotly plotly
 RUN conda update conda --all
+RUN conda install jupyterhub=2 jupyterhub-base=2 jupyter_server=1
```

Comment:
I think this is the key, for some reason they are being upgraded during the pervious step so I have to pin them back from here

jupyterlab was not working because of 

> You must use Jupyter Server v1 to load JupyterLab as notebook extension. You have v2.4.0 installed.